scripts/servicenow-bootstrap/src/servicenow_bootstrap/create_pc_refresh_service_catalog_item.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3
+"""
+ServiceNow Service Catalog Automation Script
+Creates the PC Refresh service catalog item with all necessary configurations.
+"""
+
+import argparse
+import json
+from typing import Any, Dict, List, Optional
+
+import requests
+
+from .utils import get_env_var
+
+
+class ServiceNowCatalogAutomation:
+    def __init__(self, config: Dict[str, Any]):
+        # Get sensitive values from environment variables
+        self.instance_url = get_env_var("SERVICENOW_INSTANCE_URL").rstrip("/")
```

Comment:
Would it make senbse to try to pull some of this common code out into a helper to reduce the amount of duplicate code?